<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScene</class>
 <widget class="QWidget" name="StartScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>13</pointsize>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="programName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>32</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Account Manager</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttons">
    <item>
     <widget class="QPushButton" name="logInButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signUpButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
